mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/LazyChannelPool.java

```diff
@@ -0,0 +1,100 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
```

Comment:
Shouldn't we have a reference to the licence of reactor-rabbitmq ?

Reply:
AFAIK, it's apache 2 too

Reply:
it was Apache 2 but with a different copy right of Pivotal Software. I just rebased to get that original license

Reply:
Maybe just rewrite your commit to avoid the add and delete

Reply:
yeah that was what I did